### DISCUSSION TITLE
Weird issue with Socialite: Undefined property: Laravel\Socialite\SocialiteManager::$container

### DISCUSSION THREAD

**Discussion by tamkeen-tms** [laravel/framework#33086] [General] [answered]
I am using Socialite (4.4) for social login in my app. And i get this error from inside the `SocialiteManager` itself; it cannot find the ->container!

```
Undefined property: Laravel\Socialite\SocialiteManager::$container
```
At `laravel\socialite\src\SocialiteManager.php` line 55.

I tried to set the `container` in a __construct, to app(), and it worked, but editing the library code doesn't fix the issue of course. What should I do here?

Thanks for the help.

**Comment by driesvints** [comment, marked as answer]
This is apparently because of https://github.com/laravel/socialite/pull/438. Socialite 4.x still supports Laravel 5.7 & 5.8 which don't have the container property on that class. We'll have to revert the changes. I'll take care of that today.
